django_pin_auth/forms.py

```diff
@@ -0,0 +1,76 @@
+from django.apps import apps
+from django import forms
+from django.contrib.auth import get_user_model
+from django.conf import settings
+from django.utils.safestring import mark_safe
+from django.core.urlresolvers import reverse
+from django.core.mail import EmailMessage
+from django.template.loader import render_to_string
+from django.contrib.sites.shortcuts import get_current_site
+from password_generator import generate
+
+from .models import SingleUseToken
+
+
+class RegisterForm(forms.Form):
+    email = forms.EmailField(label='Please provide a valid email')
+
+    def send_email(self, request):
+        """Send registration email."""
+        email = self.cleaned_data['email']
+        context = self._build_context(request, pin=self.token.token)
+        html = render_to_string(
+            'django_pin_auth/emails/register_body.html',
```

Comment:
For future Issue, extracting this to a `get_register_email_template` would be a nice bonus for reusability. TemplateView kind of style.

Reply:
Sure. I feel that for now, having the ability to override the template itself (normal Django stuff) should be enough first